servers/navigation_server_3d.cpp

```diff
@@ -34,6 +34,8 @@
 #include "core/config/project_settings.h"
 #endif // DEBUG_ENABLED
 
+#include "core/debugger/engine_debugger.h"
```

Comment:
Nit: Is this include being used?

Reply:
It is a leftover from an earlier iteration. Removed.